README.md

```diff
@@ -3,18 +3,20 @@
 
 # Format.cmake
 
-clang-format for CMake
+clang-format and cmake-format for CMake
 
 ## About
 
 Format.cmake adds three additional targets to your CMake project.
 
-- `format` Shows which files are affected by clang-format
-- `check-format` Errors if files are affected by clang-format (for CI integration)
-- `fix-format` Applies clang-format to all affected files
+- `format` Shows which files are affected by the format tools
+- `check-format` Errors if files are affected by the format tools (for CI integration)
+- `fix-format` Applies the format tools to all affected files
 
 To run the targets, invoke CMake with `cmake --build <build directory> --target <target name>`.
 
+To enable formatting CMake files, invoke CMake with `cmake <source directory> -DFORMAT_CHECK_CMAKE=ON`.
```

Comment:
TBH I'd rather have recommend having the dependant's CMakeLists set the option, as then collaborators and the CI will have the same formatting settings.
 
```suggestion
To enable formatting CMake files, enable the cmake option `FORMAT_CHECK_CMAKE`, e.g. by invoking CMake with `-DFORMAT_CHECK_CMAKE=ON`, or enabling the option when [adding the dependency](#how-to-integrate) (recommended).
```

And in the _How to integrate section_ add the option to `CPMAddPackage`.

```cmake
CPMAddPackage(
  NAME Format.cmake
  VERSION 1.4
  GITHUB_REPOSITORY TheLartians/Format.cmake
  # enable cmake formatting
  OPTIONS "FORMAT_CHECK_CMAKE ON"
)
```